chall02/dalba-de.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+dictionary = {"a": ".-", "b": "-...", "c": "-.-.", "d": "-..", "e": ".", "f": "..-.", 

```

Comment:
You could have made a dictionary with only lowercase or uppercase and later use `.lower()` or `.upper()` on your string